how_to_spin_up_an_htap_database_in_5_minutes_with_tidb_tispark.md

```diff
@@ -0,0 +1,251 @@
+---
+title: How To Spin Up an HTAP Database in 5 Minutes with TiDB + TiSpark
+author: ['Queeny Jin']
+date: 2018-06-08
+summary: In this 5-minute tutorial for beginners, we will show you how to spin up a standard TiDB cluster using Docker Compose on your local computer, so you can get a taste of its hybrid power, before using it for work or your own project in production.
+tags: ['TiDB', 'TiSpark', 'HTAP', 'Tutorial']
+---
+
+[TiDB](http://bit.ly/tidb_repo_publication) is an open-source distributed Hybrid Transactional and Analytical Processing (HTAP) database built by PingCAP, powering companies to do real-time data analytics on live transactional data in the same data warehouse -- no more ETL, no more T+1, no more delays. More than 200 companies are now using TiDB in production. Its 2.0 version was launched in late April 2018 (read about it in [this blog post](http://bit.ly/tidb_2_0)).
+
+In this 5-minute tutorial, we will show you how to spin up a standard TiDB cluster using Docker Compose on your local computer, so you can get a taste of its hybrid power, before using it for work or your own project in production. A standard TiDB cluster includes TiDB (MySQL compatible stateless SQL layer), [TiKV](http://bit.ly/tikv_repo_publication) (a distributed transactional key-value store where the data is stored), and [TiSpark](https://github.com/pingcap/tispark) (an Apache Spark plug-in that powers complex analytical queries within the TiDB ecosystem). 
+
+> NOTE: This guide is for Mac users; we will release a guide for Linux and Windows users soon!
+
+Ready? Let’s get started!
+
+# Setting Up:
+
+Before we start deploying TiDB, we’ll need a few things first: `Brew`, `wget`, Docker, and MySQL. If you don’t have them installed already, here are the instructions to get them.
+
+1. To install `Brew`, go [here](https://brew.sh/) .
+
+2. To install `wget`, use the command below in your Terminal: 
+
+    ```bash
+    brew install wget --with-libressl
+
+    ```
+
+3. Install Docker: [https://www.docker.com/community-edition](https://www.docker.com/community-edition).
+
+4. Install MySQL: [https://dev.mysql.com/downloads/mysql/](https://dev.mysql.com/downloads/mysql/).
+
+# Spin up a TiDB cluster
+
+Assuming you’ve installed Docker on your laptop without a hitch, let’s deploy TiDB.
+
+1. Clone TiDB Docker Compose onto your laptop:
+
+	```bash
+	git clone https://github.com/pingcap/tidb-docker-compose
```

Comment:
Assumes the user has `git` installed. I don't think it's there by default on Macs, is it? Maybe we should tell them to install it in the `brew install` command above?